common/djangoapps/edxmako/makoloader.py

```diff
@@ -22,6 +21,7 @@ class MakoLoader(object):
     """
 
     is_usable = False
+    supports_recursion = False
```

Comment:
What does this line do?  I can't find this string anywhere else in the platform code.

Reply:
hm, seems like it's used internally by django, but apparently not well documented.

Reply:
I had to add that because Django expects it to be available on Loader classes starting in 1.9: https://github.com/django/django/blob/stable/1.9.x/django/template/loaders/base.py#L101 .  I think this is the specific use of it that was blowing up until I added it: https://github.com/django/django/blob/stable/1.9.x/django/template/loader_tags.py#L111